actionkit/api/event.py

```diff
@@ -68,11 +61,11 @@ def idfromurl(path):
         if user_id:
             result = self.client.get(
                 '%s/rest/v1/eventsignup/?user=%s&%s' % (
-                    self.base_url, user_id, encode_url(query_params)))
+                    self.base_url, user_id, urlencode(query_params)))
```

Comment:
beter just to use request.get's `params` keyword:
https://realpython.com/python-requests/#query-string-parameters

(I think it wasn't always available in python2 or something, but it's definitely there now)

Reply:
++